README.md

```diff
@@ -4,16 +4,107 @@
 [![Build Status](https://travis-ci.org/jboss-fuse/vscode-datavirt.svg?branch=master)](https://travis-ci.org/jboss-fuse/vscode-datavirt)
 [![License](https://img.shields.io/badge/license-Apache%202-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+## Prerequisites
+
+- recent Visual Studio Code
+
+
 ## Features
 
-...
+Red Hat Integration - Data Virtualization Tooling enables you to create a Virtual Database definition file (VDB) and to visualize and manipulate its contents inside [VS Code](https://code.visualstudio.com/).
 
+### The Data Virtualization view
+The extension provides its own view to [VS Code](https://code.visualstudio.com/) which is called `Data Virtualization`.
 
-## How to contribute
+![Data Virtualization View](./images/dv_view.png)
 
-See [Contributing guide](./Contributing.md)
+### Creating a Virtual Database definition
 
+There are two ways to create a new VDB file. 
 
-## Prerequisites
+Option 1 would be to open up the command palette (`F1`) of [VS Code](https://code.visualstudio.com/) and to enter `New Virtual Database` into the search field like shown in below screenshot.
 
-- recent Visual Studio Code
+![Create new VDB via Command Palette](./images/dv_new_vdb_cmd_palette.png)
+
+Option 2 would be to use the contextual menu inside the `Data Virtualization View` like shown in the below screenshot.
+
+![Create new VDB via Context Menu](./images/dv_new_vdb_context_menu.png)
+
+Both options will ask you for a `name` and then provide the new VDB yaml file inside the `Explorer` view. Additionally the `Data Virtualization` view will also display the new VDB as an entry inside that view.
```

Comment:
the specific terminology of `Explorer` is a `Tree View Container` https://code.visualstudio.com/api/extension-capabilities/extending-workbench

Using the same terminology 'view' for both "tree view" and "tree view container" is ambiguous.

Reply:
fine to use non-official but widely in other extensions the "view" terminology. See Brian comment https://github.com/jboss-fuse/vscode-datavirt/pull/143#issuecomment-620615556